components/avatar/avatar.css

```diff
@@ -0,0 +1,62 @@
+@import url("/index.css");
+@import url("/components/sidebar.css");
```

Comment:
sidebar related styles can be separated into different file
In avatar.css just keep styles related to avatar only